src/main/java/uk/gov/companieshouse/uri/web/model/PreviousName.java

```diff
@@ -1,11 +1,14 @@
 package uk.gov.companieshouse.uri.web.model;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public class PreviousName {
 
     String ceasedOn;
     
     String name;
 
+    @JsonProperty("CONDate")
```

Comment:
What does CON mean?

Reply:
It stands for Change Of Name. https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/809682/uniformResourceIdentifiersCustomerGuide.pdf